meta.go

```diff
@@ -461,32 +504,34 @@ func (m Meta) MarshalYAML() (interface{}, error) {
 	}
 
 	return struct {
-		Name           string                           `yaml:"name"`
-		Summary        string                           `yaml:"summary"`
-		Description    string                           `yaml:"description"`
-		Provides       map[string]marshaledRelation     `yaml:"provides,omitempty"`
-		Requires       map[string]marshaledRelation     `yaml:"requires,omitempty"`
-		Peers          map[string]marshaledRelation     `yaml:"peers,omitempty"`
-		ExtraBindings  map[string]interface{}           `yaml:"extra-bindings,omitempty"`
-		Categories     []string                         `yaml:"categories,omitempty"`
-		Tags           []string                         `yaml:"tags,omitempty"`
-		Subordinate    bool                             `yaml:"subordinate,omitempty"`
-		Series         []string                         `yaml:"series,omitempty"`
+		Name          string                       `yaml:"name"`
+		Summary       string                       `yaml:"summary"`
+		Description   string                       `yaml:"description"`
+		Provides      map[string]marshaledRelation `yaml:"provides,omitempty"`
+		Requires      map[string]marshaledRelation `yaml:"requires,omitempty"`
+		Peers         map[string]marshaledRelation `yaml:"peers,omitempty"`
+		ExtraBindings map[string]interface{}       `yaml:"extra-bindings,omitempty"`
+		Categories    []string                     `yaml:"categories,omitempty"`
+		Tags          []string                     `yaml:"tags,omitempty"`
+		Subordinate   bool                         `yaml:"subordinate,omitempty"`
+		Series        []string                     `yaml:"series,omitempty"`
+		// Device         map[string]Device                `bson:"device,omitempty"`
```

Comment:
Uncomment

Reply:
I am not sure where this MarshalYAML is used. And I can't find the `Storage` here. Should I remove here or uncomment?

Reply:
I am not sure either, but let's add storage as a drive by fix and make it complete